content/docs/goFSH/running/_index.md

```diff
@@ -66,3 +66,24 @@ GoFSH can still generate FSH when dependencies are omitted, but the resulting FS
 ## GoFSH Outputs
 
 GoFSH populates an output directory, called **gofsh** by default. This directory will contain an **input/fsh** directory. The **input/fsh** directory will contain several **.fsh** files organized according to the `file-per-definition` style described [above](#style). Additionally, the **input/fsh** directory will contain an **index.txt** file which describes which file contains each definition. The **gofsh** directory will also contain a **sushi-config.yaml** file. If the input to GoFSH includes an ImplementationGuide resource, it is used to generate the configuration. Otherwise the configuration is generated by inferring values from the input, or by using sensible defaults if no values can be inferred. These definitions and configuration are ready to use with SUSHI, all you have to do is [run SUSHI](/docs/sushi/running).
+
+## GoFSH Best Practices
+
+There are a few best practices that GoFSH users should follow in order to achieve the best results.
+
+* For best results, run GoFSH in `json-only` mode (which is the default) against FHIR definitions that are complete (i.e., have snapshots) and have been validated. This might include:
+  * a package downloaded from an IG or FHIR registry (after unzipping it),
+  * a package in your local FHIR cache, or
+  * an IG project's `output` folder after it has been built by the IG Publisher.
+* If running GoFSH against a fully built package is not possible, run GoFSH against only the `input` folder, rather than against the entire IG source folder.
```

Comment:
```suggestion
* If running GoFSH against a fully built package is not possible, run GoFSH against only the **input** folder, rather than against the entire IG source folder.
```